client/app/dedicatedCloud/user/disable/dedicatedCloud-user-disable.controller.js

```diff
@@ -1,16 +1,34 @@
 angular
   .module('App')
-  .controller('DedicatedCloudUserDisableCtrl', ($scope, $stateParams, $translate, DedicatedCloud) => {
-    $scope.user = $scope.currentActionData;
-    $scope.disableUser = function () {
-      $scope.resetAction();
-      DedicatedCloud.disableUser($stateParams.productId, $scope.user.userId).then(
+  .controller('DedicatedCloudUserDisableCtrl', class {
```

Comment:
You could've extracted the `class` `export` and name it and declare it as an `angular` dependency at the end of the file (or even modularize it 😄 )

Reply:
But if I do that, @cbourgois won't have to do it, I don't want to spoil him